### ISSUE TITLE
Device not in Device Manager nothing happened

### ISSUE THREAD

**Issue by Progamertoat** [VirtualDisplay/Virtual-Display-Driver#77]
Also when installing this shows 
![image](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/125738008/a8203462-2473-43a5-91a4-af558d6a4e90)

meaning it dosent have a digital signature

**Comment by bud3699** [comment]
Did you follow the instructions on installing the cert. and you downloaded from the releases instead of the source right?

**Comment by Progamertoat** [comment]
You mean to put it into the c:// directory  and executing the .bat as admin?
________________________________
Von: Bud ***@***.***>
Gesendet: Samstag, 27. April 2024 20:59
An: itsmikethetech/Virtual-Display-Driver ***@***.***>
Cc: Progamertoat ***@***.***>; Author ***@***.***>
Betreff: Re: [itsmikethetech/Virtual-Display-Driver] Device not in Device Manager nothing happened (Issue #77)


Did you follow the instructions on installing the cert. and you downloaded from the releases instead of the source right?

**Comment by zjoasan** [comment]
yes, te releaase version you can download has to be uzipped into c:\ . then it creates the correct file structure to work. Start the command prompt as administrator:

cd C:\IddSampleDriver
./installCert.bat
exit

Install the driver via "Device manager" Action, Add legacy hardware, browse to C:\IddSampleDriver accept. This should show driver and be obvious what to do next to install it.

**Comment by Progamertoat** [comment]
I did EXACTLY that but It wouldnt workAm 30.04.2024 21:11 schrieb Joakim Sandström ***@***.***>:
yes, te releaase version you can download has to be uzipped into c:\ . then it creates the correct file structure to work. Start the command prompt as administrator:
cd C:\IddSampleDriver
./installCert.bat
exit
Install the driver via "Device manager" Action, Add legacy hardware, browse to C:\IddSampleDriver accept. This should show driver and be obvious what to do next to install it.

**Comment by zjoasan** [comment]
And you don't have a new device under display, after completing your installation? Beena while since i did this but i'll install the reöease once more to make sure i give the correct steps. No problems installing the certificate as I understand? Should have looked like this:

'''
C:\Windows\System32>cd \

C:\>cd IddSampleDriver

C:\IddSampleDriver>installCert.bat
root "Betrodda rotcertifikatutfärdare"
Signature matches Public Key
Certificate "Virtual Display Driver" added to store.
CertUtil: -addstore command completed successfully.
TrustedPublisher "Betrodda utgivare"
Signature matches Public Key
Certificate "Virtual Display Driver" added to store.
CertUtil: -addstore command completed successfully.
Press any key to continue . . .
'''
Device manager-> action -> add legacy hardware
Install manually
Chose Display
Driver Disk button
browse to c:\iddampledriver
Mark the iddsampledriver.inf file and chose open
click ok
mark the driver and click next,next and finnish.

During last next and finnish buttons our actual display might "flimmer", indicateing that windwos has activated the VDD.

Hope this helps

**Comment by smokybroom** [comment]
Mine shows up in device manager but not display options. Also i didnt used to have this issue and now i do, plus when i remove and reinstall the driver i no longer get the flimmer which is weird.
